README.md

```diff
@@ -39,73 +38,70 @@ The package supports:
     - [Typescript](https://github.com/vazco/meteor-universe-i18n/#typescript)
   - [Usage](https://github.com/vazco/meteor-universe-i18n/#usage)
     - [Setting/getting locale](https://github.com/vazco/meteor-universe-i18n/#settinggetting-locale)
-    - [Adding translations by methods](https://github.com/vazco/meteor-universe-i18n/#adding-translations-my-methods)
+    - [Adding translations by methods](https://github.com/vazco/meteor-universe-i18n/#adding-translations-by-methods)
     - [Getting translations](https://github.com/vazco/meteor-universe-i18n/#getting-translations)
-    - [Creating a React component](https://github.com/vazco/meteor-universe-i18n/#creating-react-component)
-    - [Formatting numbers](https://github.com/vazco/meteor-universe-i18n/#formatting-numbers)
   - [Translations files](https://github.com/vazco/meteor-universe-i18n/#translations-files)
     - [Recognition locale of translation](https://github.com/vazco/meteor-universe-i18n/#recognition-locale-of-translation)
     - [Namespace](https://github.com/vazco/meteor-universe-i18n/#namespace)
       - [Translation in packages](https://github.com/vazco/meteor-universe-i18n/#translation-in-packages)
       - [Translation in application](https://github.com/vazco/meteor-universe-i18n/#translation-in-application)
   - [API](https://github.com/vazco/meteor-universe-i18n/#api)
-  - [Blaze support](https://github.com/vazco/meteor-universe-i18n/#blaze-support)
+  - [Integration with React](https://github.com/vazco/meteor-universe-i18n/#react-integration)
+    - [Creating a React component](https://github.com/vazco/meteor-universe-i18n/#creating-react-component)
+  - [Integration with Blaze](https://github.com/vazco/meteor-universe-i18n/#blaze-integration)
   - [Integration with SimpleSchema](https://github.com/vazco/meteor-universe-i18n/blob/master/README.md#integration-with-simpleschema-package)
-  - [Supported locales](https://github.com/vazco/meteor-universe-i18n/#supported-locales)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-
 ## Installation
+
 ```sh
 $ meteor add universe:i18n
 ```
 
 ### Typescript
+
 ```sh
 $ meteor npm install --save @types/meteor-universe-i18n
 ```
 
 ## Usage
-The package is dedicated to work with React but you can use it without it as well.
 
 ```
 import i18n from 'meteor/universe:i18n';
 ```
 
-### Tutorials
-
-- [Meteor Guide](https://guide.meteor.com/ui-ux.html#universe-i18n)
-- [Internationalizing Meteor with Universe:i18n](http://sonicviz.com/2016/10/23/internationalizing-meteor/)
-
 ### Other links:
+
 - [mock universe i18n](https://github.com/logankoester/mock-meteor-universe-i18n)
 
 ### Setting/getting locale
 
 ```js
-i18n.setLocale('en-US', params)
-i18n.getLocale() //en-US
+i18n.setLocale("en-US", params);
```

Comment:
I'd rather keep this file in align with the codebase. I guess we could use Prettier here, just like for uniforms.